### ISSUE TITLE
Multiple dashboard redesign

### ISSUE THREAD

**Issue by stefwalter** [cockpit-project/cockpit#1726]
Cockpit top level navigation needs some tweaks.

 * Support multiple dashboards

Some secondary goals:

 * First time accessing cockpit, arrive at server summary, not a dashboard
 * Have a top level navbar across all parts of Cockpit

**Comment by stefwalter** [comment]
Basic design proposal:

![navigation](https://cloud.githubusercontent.com/assets/795070/5897605/08e1f4b8-a546-11e4-9a9d-e9f718bae3c4.png)

![navigation-server](https://cloud.githubusercontent.com/assets/795070/5897606/0e2661c0-a546-11e4-8293-6724646f9422.png)

**Comment by stefwalter** [comment]
The set of dashboard top navbar items moves to the right when you go into a machine. It moves back to the left, when you go to one of the dashboards. It doesn't move as long as your mouse is over the navbar.

**Comment by stefwalter** [comment]
Blocks on #1544

**Comment by andreasn** [comment]
I am starting to come to peace with this idea bit by bit :)
So the Journal would be in two places? Is one a single-machine journal and one an aggregated journal?
Does a machine view stay on, say Storage if I switch to another machine and switch back? (Im trying to figure out how much work the use case where you set up a network disk on one machine and then go to another machine and mount it there).

**Comment by stefwalter** [comment]
> So the Journal would be in two places? Is one a single-machine journal and one an aggregated journal?

Nope. sorry for that in the mockups. It shouldn't be in the sidebar.

> Does a machine view stay on, say Storage if I switch to another machine and switch back? (Im trying to figure out how much work the use case where you set up a network disk on one machine and then go to another machine and mount it there).

That could be decided. The PatternFly host switcher pattern does stay on Storage when you switch. However this is a separate issue, and orthogonal to this redesign. Whatever we choose here, it's easy to use multiple browser tabs or windows to perform two actions on two machines at once. You should be able to middle click on things to open stuff in new tabs.

**Comment by andreasn** [comment]
> Nope. sorry for that in the mockups. It shouldn't be in the sidebar.
Sounds good

> However this is a separate issue, and orthogonal to this redesign
Mostly trying to figure that out since it could regress compared to our current ui.

**Comment by sub-mod** [comment]
@stefwalter what is the update on this ? Does this effect us ?

**Comment by stefwalter** [comment]
It needs to land before we do a proof of concept, so that the Kubernetes stuff can move out of the 'Tools' menu.

But other than that, it does not affect our implementation of the kubernetes dashboard.

**Comment by stefwalter** [comment]
After implementing it, the following issues need tweaking:

 * After going to a dashboard, it should be easier to get back to servers than going through the Machines dashboard
 * The top bar is very empty in the default configuration.

Will iterate to fix these issues.
